zebra-chain/src/work/difficulty.rs

```diff
@@ -280,7 +280,16 @@ impl ExpandedDifficulty {
             Network::Testnet => (U256::one() << 251) - 1,
         };
 
-        limit.into()
+        // `zcashd` converts the PoWLimit into a compact representation before
+        // using it to perform difficulty filter checks.
+        //
+        // The Zcash specification converts to compact for the default difficulty
+        // filter, but not for testnet minimum difficulty blocks. (ZIP 205 and
```

Comment:
The minimum difficulty rule doesn't affect the value of `nBits`:
> As before, the `nBits` field of a minimum-difficulty block is still computed according to the original difficulty adjustment algorithm, and only this field is used for the purpose of computing the MeanTarget values from which subsequent difficulty changes are calculated.

ZIP 205 says that the target threshold is set to the value of PoWLimit for testnet. So I think this is well-specified; let me know if you disagree.

Reply:
Oh, I see: https://github.com/ZcashFoundation/zebra/issues/1277

Hmm, that's technically a bug in zcashd :-/